charts/lagoon-core/ci/linter-values.yaml

```diff
@@ -15,6 +15,7 @@ s3BAASAccessKeyID: baas
 s3BAASSecretAccessKey: baas123
 lagoonUIURL: http://ui:9101
 lagoonWebhookURL: http://webhook:11213
+defaultIngressClassName: nginx
```

Comment:
IMO this should either be removed or one of the ingresses should be enabled. As it is, there is no point having this value here because it has no effect and could cause confusion later.